opl/investigator/config.py

```diff
@@ -75,6 +75,10 @@ def load_config(conf, fp):
         assert not conf.history_es_server.endswith("/")
         conf.history_es_index = data["history"]["es_index"]
         conf.history_es_query = data["history"]["es_query"]
+        if "es_server_user" in data["history"]:
+            conf.history_es_server_user = data["history"]["es_server_user"]
+            conf.history_es_server_pass_env_var = data["history"]["es_server_pass_env_var"]
+            conf.history_es_server_verify = data["history"]["es_server_verify"]
```

Comment:
Pls make this option optional and True by default. Also move it out of this `if ...` as it can be used no matter if auth is used.